zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/impl/SQLPPGenerator.scala

```diff
@@ -355,6 +355,11 @@ class SQLPPGenerator extends AsterixQueryGenerator {
     }
   }
 
+  /**
+    * Append a new line and queryStr to the queryBuilder if queryStr is not empty
```

Comment:
we don't need to repeat the function name, right? 
it's better to specify the behind-scene reason  about why we don't append if the querystring is empty?